.artifakt/init-host.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+echo ">>>>>>>>>>>>>> START CUSTOM INIT SCRIPT <<<<<<<<<<<<<<<<< "
+echo "------------------------------------------------------------"
+echo "The following build args are available:"
+env
```

Comment:
Should we continue to print env vars in logs as we can't hide sensitive env vars when it's printed in custom scripts ?

Reply:
see my comment in #5 about same topic

Reply:
As discussed, to be changed in a future PR: add comments to show that env vars are available, without displaying them